cmd/cosign/cli/verify/verify.go

```diff
@@ -172,6 +173,9 @@ func (c *VerifyCommand) Exec(ctx context.Context, images []string) (err error) {
 	keyRef := c.KeyRef
 	certRef := c.CertRef
 
+	if !c.IgnoreSCT {
+		co.CTLogPubKeys, err = ctl.GetCTLogPubs(ctx)
```

Comment:
check err here?

Reply:
oops, I had forgot to push the changes when I took out the c.IgnoreSCT (See the comment above about the need for it).